src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.NetCore.cs

```diff
@@ -0,0 +1,47 @@
+#if NETCOREAPP
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Datadog.Trace.ClrProfiler.Managed.Loader
+{
+    /// <summary>
+    /// A class that attempts to load the Datadog.Trace.ClrProfiler.Managed .NET assembly.
+    /// </summary>
+    public partial class Startup
+    {
+        internal static System.Runtime.Loader.AssemblyLoadContext DependencyLoadContext { get; } = new ManagedProfilerAssemblyLoadContext();
+
+        private static string ResolveManagedProfilerDirectory()
+        {
+            string tracerFrameworkDirectory = "netstandard2.0";
+            var tracerHomeDirectory = Environment.GetEnvironmentVariable("DD_DOTNET_TRACER_HOME") ?? string.Empty;
+            return Path.Combine(tracerHomeDirectory, tracerFrameworkDirectory);
+        }
+
+        private static Assembly AssemblyResolve_ManagedProfilerDependencies(object sender, ResolveEventArgs args)
+        {
+            var assemblyName = new AssemblyName(args.Name);
+            var path = Path.Combine(ManagedProfilerDirectory, $"{assemblyName.Name}.dll");
+
+            if (assemblyName.Name.StartsWith("Datadog.Trace", StringComparison.OrdinalIgnoreCase)
+                && assemblyName.FullName.Contains("PublicKeyToken=def86d061d0d2eeb")
```

Comment:
```suggestion
                && assemblyName.FullName.Contains("PublicKeyToken=def86d061d0d2eeb", StringComparer.OrdinalIgnoreCase)
```

Reply:
Fixed in b9c80c289bb5e108f1c25d9973cb6f669659f975 . It turns out `string.Contains` doesn't have a built-in method that takes a `string` and a `StringComparer` so I had to resort to calling `string.IndexOf` which does take those arguments

Reply:
Oops, my bad! I was getting Intellisense for `IEnumerable<char>.Contains(...)` 🤦‍♂